tests/unit/test_tfidf_utils.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
also, it would be great to add an integration test for the notebook  https://github.com/microsoft/recommenders/tree/master/tests#how-to-create-tests-on-notebooks-with-papermill

If you don't have bandwidth, at least can you please mark it as todo with `@pytest.mark.skip(reason="TODO: Implement this")`

Reply:
I have created a new integration test file `./tests/integration/test_covid.py`. It does not currently have any tests to run, but I hope to write them out today. Will update.

Reply:
Unfortunately did not have time to write out the tests today, but a placeholder file has been created `./tests/integration/test_covid.py`